lib/src/verison_constants.dart

```diff
@@ -0,0 +1,17 @@
+class VersionConstants {
```

Comment:
This probably won't be required in single/multiple account release.

Reply:
yes its update